packages/core/src/main/conversation/ConversationService.ts

```diff
@@ -89,6 +89,11 @@ import type {
   TextMessage,
 } from './message/OtrMessage';
 
+interface MessageSendingCallbacks {
+  onStart: (message: GenericMessage) => void;
+  onSuccess: (message: GenericMessage) => void;
```

Comment:
Let's make them optional :)
```suggestion
  onStart?: (message: GenericMessage) => void;
  onSuccess?: (message: GenericMessage) => void;
```